jgrapht-core/src/main/java/org/jgrapht/alg/shortestpath/GraphMeasurer.java

```diff
@@ -158,6 +158,31 @@ public double getRadius()
         return graphPeriphery;
     }
 
+    /**
+     * Compute the graph pseudo-periphery.
+     * The pseudo-periphery of a graph is the set of all pseudo-peripheral vertices.
+     * A vertex u is pseudo-peripheral, if for each vertex v with $d(u,v) = \epsilon(u)$ holds $\epsilon(u) = \epsilon(v)$.
+     *
+     * @return the graph pseudo-periphery
+     */
+    public Set<V> getGraphPseudoPeriphery()
+    {
+        computeEccentricityMap();
+        Set<V> graphPseudoPeriphery = new LinkedHashSet<>();
+        ToleranceDoubleComparator comp = new ToleranceDoubleComparator(0.000000001);
```

Comment:
Use ToleratnceDoubleComparator.DEFAULT_EPSILON

Reply:
I made the change in all 3 methods that used 0.000000001 instead of  ToleranceDoubleComparator.DEFAULT_EPSILON.